_config.yml

```diff
@@ -6,7 +6,7 @@ kramdown:
 markdown:
   kramdown
 
-baseurl: /grpc.github.io
```

Comment:
It was a mistake to set it to /grpc.github.io. GitHub pages supports "subprojects" as directories, and Jekyll templates generally use that to prefix their content. This is the main page of the website, so it should not be prefixed. It just happens to sometimes work because grpc.io will be served on grpc.io/grpc.github.io and apparently will be redirected to the right content eventually, but on https it breaks the website for some reason.